### PR TITLE
Improve FAQ

### PR DESCRIPTION
**Pull request by avital**
I fixed some formatting issues, clarified some answers, and removed questions whose answers weren't really helpful.

I think the FAQ could still use some more love though...

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -22,14 +24,17 @@ submodules.
 of `init`/`apply` methods?
 
 **Answer:** Modules have an `init` and `call` function that can be used if you
-need use them. So you can do things like `Dense.call(params, X, ...)`.
+need use them. So you can do things like `Dense.call(params, X, ...)`. `Model` just
```
Comment (marcvanzee) [inline docs/faq.md]: "need use them" -> "need to use them"?

File: docs/faq.md [hunk 2]
```diff
@@ -242,9 +232,10 @@ precision in cases like batchnorm.
 
 --- 
 
-**Question: Do FLAX models have static or dynamic shapes?**
+**Question: Do Flax models have static or dynamic shapes?**
 
 **Answer:** They have static shapes. A model is created from an initial shape, 
-and it is not directly possible to change this.
+and it is not directly possible to change this. (This is a limitation of XLA
+and thus of JAX)
```
Comment (marcvanzee) [inline docs/faq.md]: Add period.

File: docs/faq.md (hunk 1 above)
Comment (avital) [reply docs/faq.md]: Done

File: docs/faq.md (hunk 2 above)
Comment (avital) [reply docs/faq.md]: Done

**Pull request merged**